src/card/aggregate-status/aggregate-status-card.html

```diff
@@ -13,8 +13,15 @@ <h2 class="card-pf-title">
       <span class="card-pf-aggregate-status-title">{{$ctrl.status.title}}</span>
     </span>
   </h2>
-  <div class="card-pf-body">
-    <p class="card-pf-aggregate-status-notifications">
+  <div class="card-pf-body" ng-class="{'show-spinner': $ctrl.showSpinner}" ng-style="$ctrl.showSpinner ? $ctrl.spinnerHeight : {}">
```

Comment:
I would expect the height to be set on the card, not on the body. It is called spinnerCardHeight and given the title height could potentially change based on knowing the title prior to data load it would be better to set the full card height.